### PR TITLE
Update Assignment3, Assignment4 and HandsOn Group11

### PR DESCRIPTION
**Pull request by ruliii**

### CODE REVIEW

**Comment by ocorcho** [comment]
Assignment 3 is ok. Now @dachafra to check assignment 4.

File: Assignment4/ruliii-w140455/task06.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+"""**TASK 6.1: Create a new class named "University"**"""
+g.add((ns.University, RDF.type, RDFS.Class))
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0/")
+janeSmithUri=ns.JaneSimth
+g.add((ns.Researcher,ns.ResearcherName,ns.JaneSmith))
```
Comment (dachafra) [inline Assignment4/ruliii-w140455/task06.py:40]: this is not correct, Jane Smith should be declared as a resource and should belong to the class researcher

File: Assignment4/ruliii-w140455/task06.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
+g.add((janeSmithUri,VCARD.FN,Literal("Jane Smith")))
+g.add((janeSmithUri,VCARD.Family,Literal("Smith")))
+g.add((janeSmithUri,VCARD.Given,Literal("Jane")))
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+g.add((ns.JohnSmith,ns.WorksIn,ns.UPM))
```
Comment (dachafra) [inline Assignment4/ruliii-w140455/task06.py:48]: and UPM is??? is not a university?

File: Assignment4/ruliii-w140455/task07.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+for r in g.query(q2):
+  print("SPARQL: ",r)
+
+for t in g.triples((None,RDF.type,ns.Person)):
+  print("RDFLib:",t)
+"""**TASK 7.3: List all individuals of "Person" and all their properties including their class with RDFLib and SPARQL**"""
+
+q3 = prepareQuery('''
```
Comment (dachafra) [inline Assignment4/ruliii-w140455/task07.py:63]: this is not correct, you should retrieve all the triples belong to each individual annotated as person

**Review by dachafra — CHANGES_REQUESTED** [review]

File: Assignment4/ruliii-w140455/task06.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+"""**TASK 6.1: Create a new class named "University"**"""
+g.add((ns.University, RDF.type, RDFS.Class))
+
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+g.add((ns.Researcher, RDFS.subClassOf, ns.Person))
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+VCARD = Namespace("http://www.w3.org/2001/vcard-rdf/3.0/")
+g.add((ns.JaneSmith,RDF.type,ns.ResearcherName,))
```
Comment (dachafra) [inline Assignment4/ruliii-w140455/task06.py:39]: still wrong, and also bad syntaxis.

**Comment by rgcmme** [comment]
The hands-on part looks good. Once the assignments are ready it can be merged.

**Review by dachafra — APPROVED** [review]

**Comment by dachafra** [comment]
assignmet4 ok

**Pull request merged**